tasks/mpi/mezhuev_m_lattice_torus/func_tests/main.cpp

```diff
@@ -0,0 +1,254 @@
+#include <gtest/gtest.h>
+
+#include <boost/mpi/communicator.hpp>
+#include <boost/mpi/environment.hpp>
+#include <cmath>
+#include <memory>
+#include <numeric>
+#include <vector>
+
+#include "mpi/mezhuev_m_lattice_torus/include/mpi.hpp"
```

Comment:
Добавить пустую строку после
```suggestion
#include "mpi/mezhuev_m_lattice_torus/include/mpi.hpp"

```